build/post-install.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+cleanInstall() {
+    printf "\033[32m Post Install of an clean install\033[0m\n"
+    printf "\033[32m Reload the service unit from disk\033[0m\n"
+    systemctl daemon-reload ||:
+    printf "\033[32m Unmask the service\033[0m\n"
+    systemctl unmask webhook-go.service ||:
+    printf "\033[32m Set the preset flag for the service unit\033[0m\n"
+    systemctl preset webhook-go.service ||:
+    printf "\033[32m Set the enabled flag for the service unit\033[0m\n"
+    systemctl enable webhook-go.service ||:
+    systemctl restart webhook-go.service ||:
```

Comment:
do we have to autostart the service? debian does this but I am not a big fan of it. also without a user-provided config it doesnt make sense to start the service?

Reply:
It's pretty standard for packages to auto-start the service. 

I don't like it either, but I also don't want to confuse users by changing expected behavior

Reply:
Is that common anywhere except for Debian? RHEL, Gentoo and  Arch Linux dont enable/start services after the installation.

Reply:
Updated with logs on what to do instead of auto-starting